<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>bsCuttingBoxWidget</class>
  <widget class="QWidget" name="bsCuttingBoxWidget">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>384</width>
        <height>195</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Cutting Box Tool</string>
    </property>
    <layout class="QHBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>6</number>
      </property>
      <item>
        <widget class="QFrame" name="Frame3">
          <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout">
            <property name="margin">
              <number>11</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item row="0" column="2">
              <layout class="QVBoxLayout">
                <property name="margin">
                  <number>0</number>
                </property>
                <property name="spacing">
                  <number>6</number>
                </property>
                <item>
                  <layout class="QHBoxLayout">
                    <property name="margin">
                      <number>0</number>
                    </property>
                    <property name="spacing">
                      <number>6</number>
                    </property>
                    <item>
                      <widget class="QLabel" name="m_label_i">
                        <property name="minimumSize">
                          <size>
                            <width>20</width>
                            <height>0</height>
                          </size>
                        </property>
                        <property name="text">
                          <string>I</string>
                        </property>
                        <property name="alignment">
                          <set>Qt::AlignCenter</set>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QSlider" name="m_slider_i">
                        <property name="minimum">
                          <number>-100</number>
                        </property>
                        <property name="maximum">
                          <number>100</number>
                        </property>
                        <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <layout class="QHBoxLayout">
                    <property name="margin">
                      <number>0</number>
                    </property>
                    <property name="spacing">
                      <number>6</number>
                    </property>
                    <item>
                      <widget class="QLabel" name="m_label_j">
                        <property name="minimumSize">
                          <size>
                            <width>20</width>
                            <height>0</height>
                          </size>
                        </property>
                        <property name="text">
                          <string>J</string>
                        </property>
                        <property name="alignment">
                          <set>Qt::AlignCenter</set>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QSlider" name="m_slider_j">
                        <property name="minimum">
                          <number>-100</number>
                        </property>
                        <property name="maximum">
                          <number>100</number>
                        </property>
                        <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <layout class="QHBoxLayout">
                    <property name="margin">
                      <number>0</number>
                    </property>
                    <property name="spacing">
                      <number>6</number>
                    </property>
                    <item>
                      <widget class="QLabel" name="m_label_k">
                        <property name="minimumSize">
                          <size>
                            <width>20</width>
                            <height>0</height>
                          </size>
                        </property>
                        <property name="text">
                          <string>K</string>
                        </property>
                        <property name="alignment">
                          <set>Qt::AlignCenter</set>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QSlider" name="m_slider_k">
                        <property name="minimum">
                          <number>-100</number>
                        </property>
                        <property name="maximum">
                          <number>100</number>
                        </property>
                        <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </item>
            <item row="0" column="0">
              <layout class="QGridLayout">
                <property name="margin">
                  <number>0</number>
                </property>
                <property name="spacing">
                  <number>6</number>
                </property>
                <item row="1" column="0">
                  <widget class="QLabel" name="m_label_02">
                    <property name="text">
                      <string>L</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="2" column="0">
                  <widget class="QLabel" name="m_label_03">
                    <property name="text">
                      <string>H</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="1" column="1">
                  <widget class="QLineEdit" name="m_l">
                    <property name="focusPolicy">
                      <enum>Qt::ClickFocus</enum>
                    </property>
                    <property name="text">
                      <string>NULL</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="1">
                  <widget class="QLineEdit" name="m_h">
                    <property name="focusPolicy">
                      <enum>Qt::ClickFocus</enum>
                    </property>
                    <property name="text">
                      <string>NULL</string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="0">
                  <widget class="QLabel" name="m_label_01">
                    <property name="text">
                      <string>W</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1">
                  <widget class="QLineEdit" name="m_w">
                    <property name="enabled">
                      <bool>true</bool>
                    </property>
                    <property name="focusPolicy">
                      <enum>Qt::ClickFocus</enum>
                    </property>
                    <property name="text">
                      <string>NULL</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="2" column="0">
              <widget class="QPushButton" name="m_reset_button">
                <property name="text">
                  <string>Reset</string>
                </property>
              </widget>
            </item>
            <item row="1" column="0" rowspan="1" colspan="4">
              <widget class="Line" name="Line4"/>
            </item>
            <item row="0" column="1">
              <widget class="Line" name="Line5"/>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
